workflow/workflow.go

```diff
@@ -385,5 +385,46 @@ func NewWorkflowCommands() []*cli.Command {
 				}),
 			Action: TraceWorkflow,
 		},
+		{
+			Name:      "update",
+			Usage:     common.UpdateWorkflowDefinition,
+			UsageText: common.WorkflowUpdateUsageText,
+			Flags: append(common.FlagsForExecution,
+				&cli.StringFlag{
+					Name:     common.FlagUpdateWaitPolicy,
+					Usage:    "Wait policy determines which status the client should wait to receive from the server:" + strings.Join(mapKeysToArray(updateWaitPolicyMap), ", "),
+					Category: common.CategoryMain,
+					Required: true,
+					// TODO: Remove these after more wait policies has been added
+					Hidden:     true,
```

Comment:
Is my understanding correct that `Hidden: true` here will prevent this flag from be printed in the help output?

Reply:
yes, that is correct. Since right now we only have 1 wait policy, I think we have 3 options: 
- ignore wait policy altogether and implement it later 
- implement it and show the flag with a single wait policy. 
- implement it and hide the flag for now and add it later. 

WDYT?

Reply:
This sounds good to me

Reply:
I'd say the first option. It's not so hard to add later and it prevents accidental async invocation from well-intentioned code divers.

Reply:
I see, very good point. Ok then, I can remove this and `--update-id` flag which will be ready after wait policies become available via sdk.